core/common/src/main/java/alluxio/util/FormatUtils.java

```diff
@@ -198,6 +198,30 @@ public static long parseSpaceSize(String spaceSize) {
   }
 
   /**
+   * Parses a time from millisecond to target unit
+   *
+   * @param unit the target unit set by user
+   * @param ori the original time value set in millisecond
+   * @return the time value transformed to target unit
+   */
+  public static double parseTimeUnit(String unit, String ori) {
+	double time = Double.parseDouble(ori);
+    if (unit.equals("MILLISECOND")|| unit.equals("MS")){
+      return time;
+    }else if (unit.equals("SECOND")|| unit.equals("SEC") || unit.equals("S") ){
```

Comment:
please refer to "http://www.alluxio.org/docs/1.4/en/Contributing-to-Alluxio.html#coding-style" for code style